s4e-web/src/app/views/map-view/view-manager/layer-picker/layer-picker.component.ts

```diff
@@ -21,4 +21,6 @@ export class LayerPicker implements OnInit {
 
   ngOnInit() {
   }
+
```

Comment:
I would rather suggest adding

`@Input() showFavouriteButtons: boolean = false;`
 as an input.

Reply:
I think that logic is simpler if we check if the object contains the favorite field instead of adding new Inputs.